src/button/button.css

```diff
@@ -1,93 +1,204 @@
-/* "Unstyled" button css taken from https://gist.github.com/MoOx/9137295 */
-button.omrs-unstyled,
-input[type="submit"].omrs-unstyled,
-input[type="button"].omrs-unstyled {
+/* The default button styles */
+button.omrs-btn,
+input[type="submit"].omrs-btn,
+input[type="button"].omrs-btn {
   border: none;
-  margin: 0;
-  padding: 0;
-  width: auto;
-  overflow: visible;
-  height: inherit;
-  background: transparent;
-
-  /* inherit font & color from ancestor */
-  color: inherit;
-  font: inherit;
-  text-align: inherit;
-
-  /* Normalize `line-height`. Cannot be changed from `normal` in Firefox 4+. */
-  line-height: normal;
-
-  /* Corrects font smoothing for webkit */
-  -webkit-font-smoothing: inherit;
-  -moz-osx-font-smoothing: inherit;
-
-  /* Corrects inability to style clickable `input` types in iOS */
-  -webkit-appearance: none;
-}
-
-button.omrs-unstyled:focus,
-input[type="submit"].omrs-unstyled:focus,
-input[type="button"].omrs-unstyled:focus {
-  /*
-    there's some 3rd party css that turns these off using !important, so the only way to override them is to also use important.
-    this is why you don't use !important -- it makes it hard for other people to override it when they need to :(
-  */
+  font-size: 1.0625rem;
+  line-height: 1rem;
+  letter-spacing: 0.0125rem;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  vertical-align: middle;
+  align-items: center;
+  text-align: center;
+  white-space: nowrap;
+  border-radius: 0.25rem;
+  padding: 0.75rem 1.5rem;
+}
+
+button.omrs-btn > i,
```

Comment:
This is for icons?

I actually would prefer using an `svg` element for icons, or if not that then a `span`. The `<i>` element is for italicizing text, but was hijacked to try to represent icons, too. But that's the semantic meaning of the element.

Using svg's is the easiest way to do icons because you can size them properly. Icon fonts sort of suck because they're harder to size and space appropriately.

Thoughts?

Reply:
I agree.